### PR TITLE
Massmint onboarding

### LINKED ISSUE #5176: MassMint: Onboarding
* #4964 

## Figma
---
- [Detailed interactions and logic is available Figma](https://www.figma.com/file/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?node-id=0%3A1&t=ynDINS3WQGGG9Dj0-1). 
- all the needed screens
- [interactive prototype ](https://www.figma.com/proto/HCmIzbN0a2XhsOstHkLiwi/Mass-Mint---Handoff?page-id=0%3A1&node-id=1%3A332&viewport=1014%2C587%2C0.04&scaling=scale-down-width&starting-point-node-id=1%3A332&show-proto-sidebar=1)

Please take the prototype with a grain of salt, not everything is perfect and 1:1 to final design how it should look, so in case you dont know something, dont hesitate to ask @VladoTheBoi 

## Onboarding
- onboarding is displayed only once when  entering Mass Mint for the first time, 
   when entering for the 2nd time, no onboarding displayed  , user is taken straight to mass mint
- user can scroll between slides by pressing Next button or Arrows 
- user has option to go back to onboarding from mass mint feature
- button “done” is displayed on the last slide
- “skip” button is displayed on every slide except the last one 
image

@VladoTheBoi  how to navigate to this page?

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5176 
- [ ] Requires deployment <>
- [x] test on https://deploy-preview-5205--koda-nuxt.netlify.app/bsx/massmint

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Comment by daiagi** [comment]
Not quite done yet

### TODO

- [ ] translations
- [ ] save user visited status in store and respond to it
- [ ] extract card component to brik lib?

**Comment by daiagi** [comment]
Not quite done yet

### TODO

- [ ] translations
- [ ] save user visited status in store and respond to it
- [ ] extract card component to brik lib?

**Comment by daiagi** [comment]
DeepSource is complaining about this:

image


honestly I've copy-pasted index page of massmint  from explore index page, so I'm surprised it's not happy

what the problem here? what should i do to fix it?

**Comment by roiLeo** [comment]
> what should i do to fix it?

ignore them 🙈

File: stores/massmint.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { createPinia, defineStore } from 'pinia'
+import piniaPersist from 'pinia-plugin-persist'
```
[the remark this answers is not in the document]
Comment (daiagi) [reply stores/massmint.ts:2]: oh alright

**Comment by vikiival** [comment]
> @daiagi for now just add another item in create dropdown tab. Navigation then will be revised and improved in https://github.com/kodadot/nft-gallery/issues/4752

**Comment by daiagi** [comment]
@vikiival  why did you close it?

**Comment by vikiival** [comment]
> yeah ok, that's just for how to navigate to the page, doesn't mean the entire work here is not relvent

Agree!

Please open a new PR with a dropdown

**Pull request closed without merging**